README.md

```diff
@@ -15,11 +15,11 @@ For more detailed information on the server and API design refer to the [wiki](h
 | NI-Digital Pattern Driver | 20.6.0 | Not Supported | Not Supported |
 | NI-DMM                    | 20.0.0 | 20.1.0        | 20.5.0        |
 | NI-FGEN                   | 20.0.0 | Not Supported | Not Supported |
-| NI-RFmx Bluetooth         | 21.5.0 | Not Supported | Not Supported |
-| NI-RFmx LTE               | 21.5.0 | Not Supported | Not Supported |
-| NI-RFmx NR                | 21.5.0 | Not Supported | Not Supported |
-| NI-RFmx SpecAn            | 21.5.0 | Not Supported | Not Supported |
```

Comment:
Thanks, @reckenro and @astarche. I've updated the column headers, but I'm not sure what the best process is for spot-testing these as you mention. Just run the system tests on a machine with the 21.5 drivers? Add new tests? Something else..?

Reply:
Yeah, just run the system tests on a machine with the 21.5 drivers. I think you added a few tests for new methods added in 22.5? If so, great. We can confirm those fail with a reasonable error (i.e., NOT_IMPLEMENTED) just to sanity check the setup. If you're not seeing a good way to do that, or not getting the results you expect: ping me and we can figure something out.

Reply:
I only have these test failures with RFmx 21.5, none of which are related to RFmx:
```
[  FAILED  ] 3 tests, listed below:
[  FAILED  ] SessionUtilitiesServiceTests.SysCfgLibraryPresent_EnumerateDevices_DevicePropertiesIncludesNameModelVendorSerialNumberProductId
[  FAILED  ] NiDCPowerSessionTest.InitWithErrorFromDriver_ReturnsDriverErrorWithUserErrorMessage
[  FAILED  ] NiMxLcTerminalAdaptorRestrictedDriverApiTests.RefreshedTerminalCache_GetDeviceContainer_StatusOK
```

But I think that's not unexpected since the changed attributes share the same ID and some just added new enum values, so there _wouldn't_ be runtime errors. I think there would be issues compiling specific old code against new the proto files, though.

Reply:
Yeah, that looks reasonable to me. There's been some intermittency and that one at the bottom is fully failing. I think someone has a task to either add the required stuff for that test to the grpcbot or disable it on the SystemTestRunner.

I don't remember if there were any new RFmx tests added for 22.5. If so, I'd expect those to fail as well with NOT_IMPLEMENTED or something like Alex said if run against 21.5 drivers. But if there are no new tests, this seems reasonable.

Reply:
No new tests; I think at most I updated a couple of places to match renamed attributes, like [this](https://github.com/ni/grpc-device/pull/710/commits/052237b706db9279715ad8ee61b137e5dc2bc5b5).